app/views/user_case_infos/_form.html.erb

```diff
@@ -29,6 +29,11 @@
   <div class="field">
```

Comment:
We're using a `field` class on all the div/form field wrappers. With bootstrap, I've always used the `form-group` class to wrap a label & form element. `field` seems to work OK too?